descriptors/ModuleDescriptor-template.json

```diff
@@ -119,14 +119,20 @@
       "displayName": "get RM API configuration status",
       "description": "Get RM API configuration status"
     },
+    {
+      "permissionName": "kb-ebsco.provider.get",
```

Comment:
How do you feel about renaming this permissionName to "kb-ebsco.providerCollection.get" since it is a list of providers? We will have a separate endpoint to get a provider using provider id.